### PR TITLE
Apply yapf

### PR DESCRIPTION
**Pull request by hirosassa**
## What is this PR?

TSIA

### CODE REVIEW

File: yapf.ini [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [style]
```
Comment (Hi-king) [inline yapf.ini:1]: Wow! yapf.ini already exists :) (Haven't been used, though)

**Review by Hi-king — APPROVED** [review]
@hirosassa cool! LGTM

File: yapf.ini (hunk 1 above)
Comment (hirosassa) [reply yapf.ini:1]: Yeah, surprised

**Comment by hirosassa** [comment]
Thanks!

**Pull request merged**